Test/test.test.js

```diff
@@ -0,0 +1,23 @@
+const request = require('supertest');
+const app = require("../src/app");
+
+test('/movies should return the movies page', (done) => {
+    request(app)
+    .get('/movies')
+    .expect(200)
+    .expect('Content-Type', /html/)
+    .end((err, res) => {
+        if (err) return done(err);
+        done();
+    })
+})
+test('/apiMovies should return the movies page with default items', (done) => {
+    request(app)
+    .get('/apiMovies')
```

Comment:
you don't need to test this endpoint since it makes an API call, as this will make the test impure (a test that depends on an external factor is not reliable). If the external api fails or changes its state then your test will no longer work so it is not reliable.